src/client/renderer/r_framebuffer.c

```diff
@@ -24,56 +24,60 @@
 /**
  * @brief
  */
-r_framebuffer_t R_CreateFramebuffer(r_pixel_t width, r_pixel_t height, _Bool multisample) {
+static GLuint R_CreateFramebufferTexture(const r_framebuffer_t *f,
+										 GLenum attachment,
+										 GLenum internal_format,
+										 GLenum format,
+										 GLenum type) {
+	GLuint texture;
+
+	glGenTextures(1, &texture);
+	glBindTexture(GL_TEXTURE_2D, texture);
+
+	glTexImage2D(GL_TEXTURE_2D, 0, internal_format, f->width, f->height, 0, format, type, NULL);
+
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_NEAREST);
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
+	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
+
+	glBindTexture(GL_TEXTURE_2D, 0);
+
+	glFramebufferTexture2D(GL_FRAMEBUFFER, attachment, GL_TEXTURE_2D, texture, 0);
+
+	return texture;
+}
+
+/**
+ * @brief
+ */
+static GLuint R_CreateColorAttachment(const r_framebuffer_t *f, GLenum attachment) {
+	return R_CreateFramebufferTexture(f, attachment, GL_RGBA8, GL_RGBA, GL_UNSIGNED_BYTE);
+}
+
+/**
+ * @brief
+ */
+static GLuint R_CreateDepthAttachment(const r_framebuffer_t *f, GLenum attachment) {
+	return R_CreateFramebufferTexture(f, attachment, GL_DEPTH24_STENCIL8, GL_DEPTH_STENCIL, GL_UNSIGNED_INT_24_8);
+}
+
+/**
+ * @brief
+ */
+r_framebuffer_t R_CreateFramebuffer(r_pixel_t width, r_pixel_t height) {
 
 	r_framebuffer_t framebuffer = {
 		.width = width,
 		.height = height,
-		.multisample = multisample
 	};
 
 	glGenFramebuffers(1, &framebuffer.name);
 	glBindFramebuffer(GL_FRAMEBUFFER, framebuffer.name);
 
-	if (multisample) {
```

Comment:
I removed the renderbuffer support here, because to do post-processing, we need to render to textures, not renderbuffers. You can't bind and sample renderbuffers -- that's actually their point. OpenGL can make some optimizations to renderbuffers, *because* you'll never ask to read them. That's incompatible with post-effects.

It's not clear to me if, in removing this, I also broke multisample entirely? It still runs without complaining, and the game looks normal. I couldn't immediately tell if it was antialiasing or not.